GUI/Controls/GLViewerControl.cs

```diff
@@ -16,6 +16,9 @@ namespace GUI.Controls
 {
     internal partial class GLViewerControl : UserControl
     {
+        private const long TicksPerSecond = 10_000_000;
```

Comment:
Would probably be nicer to define TicksPerSecond as a function of the frequency and the stopwatch frequency, instead of the other way around